.changeset/big-starfishes-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-labeled-field": patch
```

Comment:
Note: all of the changesets are marked either a patch or minor update for the `wonder-blocks-labeled-field` package. 

Currently, the package is at [1.0.5](https://github.com/Khan/wonder-blocks/blob/3fbcb28dc9de968828d32ff55c3a2142a0df8736/packages/wonder-blocks-labeled-field/package.json#L3) and exports a string as default as a placeholder ([ref](https://github.com/Khan/wonder-blocks/blob/main/packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx)). 

I was wondering if it would be worth changing to a major version. It isn't currently used in `webapp` or `perseus` though, so it might be fine if we leave it at v1.x.x. The new version will be `1.1.0` 

Let me know if anyone has any thoughts or concerns around this!

Reply:
IMO It's fine marking it as non-breaking change as there's no current usage